component/src/main/java/io/siddhi/extension/io/file/FileSource.java

```diff
@@ -618,11 +617,23 @@ public void pause() {
     }
 
     public void resume() {
-        try {
-            updateSourceConf();
-            deployServers();
-        } catch (ConnectionUnavailableException e) {
-            throw new SiddhiAppRuntimeException("Failed to resume siddhi app runtime.", e);
+        if (dirUri != null && scheduledFuture != null) {
+            this.scheduledFuture = siddhiAppContext.getScheduledExecutorService().
+                    scheduleAtFixedRate(fileSourcePoller, 0, 1, TimeUnit.SECONDS);
+        }
+        if (isTailingEnabled && fileSourceConfiguration.getFileServerConnector() != null) {
```

Comment:
If tailing disabled, and if its a large file which will take time more than the persisting interval, the file processing may stop. Isnt it?
Shall we use [filePointer](https://github.com/siddhi-io/siddhi-io-file/blob/4858827171969d749751e3f580fa6dee853c83a8/component/src/main/java/io/siddhi/extension/io/file/FileSource.java#L951) and resolve that issue?
This may affect for the files that will be processing in a given dir.uri as well

Reply:
Hi Ramindu, 
In the existing pause() method, the file processing thread is not stopped which is the reason why I thought that there is nothing to resume in the resume() method, assuming that the file will be processed anyway. But let's check this further, to make sure that happens. I have created a separate issue to track this particular issue: https://github.com/siddhi-io/siddhi-io-file/issues/120.

Thanks